pypeclub/OpenPype
# Maya: Disable rendering of all lights for render instances submitted through Deadline.

## Brief description
Fixes #2241 by adding flag "RenderSetupIncludeLights" to deadline job information.

## Description
Disables rendering all lights in the scene to conform with render layers, such that all the lights in the scene aren't automatically rendered

## Additional info
Previously, jobs submitted through deadline would render all the lights because of the lack of the `RenderSetupIncludeLights` flag.  This PR fixes the issue by adding that flag to the project settings to be a global variable, that can be overridden through the created render instance in the `Extra Attributes` tab in the attribute editor in Maya.  

The old PR (found here: https://github.com/pypeclub/OpenPype/pull/3038) was discarded in favor of this one.


## Testing notes:
1. Create multiple lights in the scene.
2. Create geometry.
3. Create render instance selecting only one light and the geometry.
4. Create another render layer selecting another light and the geometry through the render setup panel in Maya (Windows->Rendering Editors->Render Setup).
5. Make sure the layers each contain collection that includes the geometry and the intended light, remove any wildcards (*) that exist.
6. Publish.

The jobs submitted should have the flag disabled as `False` in the `Job Properties`.
